s3watcher/lambda/package.json

```diff
@@ -4,17 +4,20 @@
   "description": "",
   "main": "S3Watcher.js",
   "dependencies": {
+    "aws-sdk": "^2.2.21",
```

Comment:
Ah OK that's why it was in dev deps. I'll remove it as it's now in the deps of the scripts package.json. Thanks!